test/chainsaw/testmetrics/dynatrace-provider/00-install.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetric
+metadata:
+  name: podtatometric
+spec:
+  provider:
+    name: "dynatrace"
+  query: "query"
+  fetchIntervalSeconds: 5
+
+---
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetricsProvider
+metadata:
+  name: dynatrace
+spec:
+  secretKeyRef:
```

Comment:
remove secret ref here since there is no secret installed line 17-18 and 19 are not needed)